crates/libtiny_tui/src/config.rs

```diff
@@ -31,6 +31,9 @@ pub(crate) struct Config {
 
     #[serde(default)]
     pub(crate) key_map: Option<KeyMap>,
+
+    #[serde(default)]
+    pub(crate) blocked_users: HashSet<String>,
```

Comment:
You will also want to add this to the [sample config file ](https://github.com/osa1/tiny/blob/master/crates/tiny/config.yml)

Reply:
Done